plugins/generic/externalFeed/classes/ExternalFeedDAO.inc.php

```diff
@@ -199,7 +210,7 @@ function deleteExternalFeedById($feedId) {
 	 * Delete external_feed by journal ID.
 	 * @param $journalId int
 	 */
-	function deleteExternalFeedsByJournalId($journalId) {
+	public function deleteExternalFeedsByJournalId($journalId) {
```

Comment:
Under the modern (and app-agnostic) conventions, functions like this would be `deleteByContextId`.